src/core/agent/services/multiSigService.ts

```diff
@@ -568,9 +589,9 @@ class MultiSigService extends AgentService {
       recp,
       {
         gid: serder.pre,
-        smids: rmids,
-        rmids: rmids,
-        rstates: states,
+        smids: smidsMember,
+        rmids: rmidsMember,
+        rstates: rstates,
```

Comment:
I don't think we need `rstates` here. The integration tests on Signify only pass `smids` and `rmids`